extra_tests/snippets/builtin_object.py

```diff
@@ -7,10 +7,6 @@ class MyObject:
 assert myobj == myobj
 assert not myobj != myobj
 
-object.__subclasshook__() == NotImplemented
-object.__subclasshook__(1) == NotImplemented
```

Comment:
Line 10 and 12 are less useful. But Line 11 still is valid.

Reply:
10 and 12 were type errors instead, which conforms to cpython a little better I think